demo/app/main.js

```diff
@@ -5,29 +5,5 @@ var Router = ReactRouter.Router;
 var hashHistory = ReactRouter.hashHistory;
 var routes = require("./routes.jsx");
 
-/**
- * Demo app used to test and demonstrate UI components
- */
-var chamel = {
+ReactDOM.render(<Router history={hashHistory} routes={routes}/>, document.querySelector("#app-main"));
```

Comment:
I removed the var chamel = {} module.exports = chamel; here because in index.html line 41, it cannot recognize the chamel.run. I always get the error chamel is not defined. been trying to fix this but no luck. May I ask if you have any idea why index.html cant see the chamel variable?